tools/dataProtector/advanced/apps-whitelist.md

```diff
@@ -42,6 +42,9 @@ this iExec TEE dApp is simple:
 
 3. Re-upload the encrypted data to IPFS and return the URL.
 
+⚠️ Please note: This application and its whitelist can only be used within the
+dataProtectorSharing module, as it is owned by the sharingContract.
+
```

Comment:
Renders as:
<img width="823" alt="Capture d’écran 2024-08-05 à 10 39 52" src="https://github.com/user-attachments/assets/05fce73b-2d31-438e-a8a6-0f4014685282">

To make it more noticeable:
<img width="946" alt="Capture d’écran 2024-08-05 à 10 54 42" src="https://github.com/user-attachments/assets/15142c01-8309-452e-8ec3-f99d80501076">